pkgs/tools/admin/ossutil/default.nix

```diff
@@ -1,32 +1,25 @@
 { lib, buildGoModule, fetchFromGitHub, fetchpatch }:
 
 buildGoModule rec {
-  version = "1.7.6";
+  version = "1.7.16";
   pname = "ossutil";
 
   src = fetchFromGitHub {
     owner = "aliyun";
     repo = "ossutil";
-    rev = version;
-    sha256 = "1hkdk0hidnm7vz320i7s4z7jngx2j70acc93agii2b3r2bb91l3d";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-J6t8QoyCvbGrUX2AkdqugztchP7Cc0jZsrn1+OB2hVY=";
   };
 
-  # this patch is required to add go mods to fetch dependencies
-  patches = [
-    (fetchpatch {
-      url = "https://github.com/aliyun/ossutil/commit/64067e979fb24ffb198a0c4eca718e81b63f514e.patch";
-      sha256 = "2pn0BcbNNL+iMema54LRpG/ca5kyDugLIZQ/TMhYG/8=";
-    })
-  ];
-
-  vendorSha256 = "lem9Jg4Ywv3qcIwhiZHNi1VH5HxxNr6mnefOLCzPL70=";
+  vendorSha256 = "sha256-oxhi27Zt91S2RwidM+BPati/HWuP8FrZs1X2R2Px5hI=";
 
   # don't run tests as they require secret access keys that only travis has
   doCheck = false;
 
   meta = with lib; {
-    homepage = "https://github.com/aliyun/ossutil";
     description = "A user friendly command line tool to access Alibaba Cloud OSS";
+    homepage = "https://github.com/aliyun/ossutil";
+    changelog = "https://github.com/aliyun/ossutil/blob/master/CHANGELOG.md";
```

Comment:
```suggestion
    changelog = "https://github.com/aliyun/ossutil/blob/v${version}/CHANGELOG.md";
```

Reply:
Good call, fixed! Thanks for the review!